jenkins/README.md

```diff
@@ -43,39 +37,104 @@ If the author is not in the whitelist but should be trusted then by adding a
 comment **add to whitelist** on the PR, the author will then be able to run the
 jobs on its own.
 
-### Cloud Resources cleanup
+## Cloud Resources cleanup
 
 There is a Jenkins [master job](https://jenkins.nordix.org/view/Airship/job/airship_master_integration_tests_cleanup/)
 that cleans up all the leftover VMs from
 [CityCloud](https://www.citycloud.com/) every 6 hours which has failed to be
 deleted at the end of v1alphaX integration test.
 
-### "Can one of the admins verify this patch?"
+## "Can one of the admins verify this patch?"
 
 For all the PRs from authors that are not whitelisted, the bot will add a
 comment "*Can one of the admins verify this patch?*". This means that the author
 is not in the whitelist and that someone from the metal3-io organization should
 review the PR and run the tests (with */test-integration*) or add the author to
 whitelist if trusted.
 
-## Jenkins configuration
+## Job logs
+
+Pods, CRDs logs are collected at the end of each Jenkins job run and
+archived so that they can be later used for debugging purposes. You can
+find the archived logs under the  "*Build artifacts*" section of the
+[job](https://jenkins.nordix.org/view/Airship/job/airship_metal3io_bmo_v1a4_integration_test_ubuntu/).
+Please note that, depending on which one happens first, logs will be removed after
+either 30 days of its life or 100 job runs, whichever occurs first.
```

Comment:
Now we have a duplication of "depending on which one happens first" and "whichever occurs first".
```suggestion
Please note that the logs will be removed 30 days after creation or after 100 subsequent job runs, whichever occurs first.
```

Reply:
ops, fixed